app/config/config.yml

```diff
@@ -6,11 +6,11 @@ imports:
 # Put parameters here that don't need to change on each machine where the app is deployed
 # http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
-    locale: en
+    locale: fr
```

Comment:
Parameters must be in parameters.dist.yml (default) and parameters.yml (custom). You project can be used by a non-french developer.

For example, locale must be 'en' in parameters.dist.yml and 'fr' in parameters.yml.